js/blocks/IntervalsBlocks.js

```diff
@@ -172,7 +172,10 @@ function setupIntervalsBlocks(activity) {
         constructor() {
             super("intervalnumber", _("interval number"));
             this.setPalette("intervals", activity);
-            this.setHelpString(_("The Interval number block returns the number of scalar steps in the current interval."));
+            this.setHelpString([_("The Interval number block returns the number of scalar steps in the current interval.") , 
+                                  "" ,
```

Comment:
Should this be "documentation",

Reply:
@walterbender sir, updated the code as suggested. Let me know if anything needs changing. Thank you